runtest

```diff
@@ -2,6 +2,8 @@
 TCL_VERSIONS="8.5 8.6"
```

Comment:
This file seems to have been committed with CRLF endings, leading to "/bin/sh^M: bad interpreter" errors. This can be fixed by running `set fileformat=unix` in vim and saving.